public/app/plugins/datasource/zipkin/datasource.ts

```diff
@@ -60,7 +60,10 @@ export class ZipkinDatasource extends DataSourceWithBackend<ZipkinQuery, ZipkinJ
     }
 
     if (target.query) {
-      const query = this.applyVariables(target, options.scopedVars);
+      if (config.featureToggles.zipkinBackendMigration && !this.nodeGraph?.enabled) {
```

Comment:
Cause if `this.nodeGraph.enabled === true`, we are adding additional response processing, that we currently don't do in backend - see https://github.com/grafana/grafana/blob/main/public/app/plugins/datasource/zipkin/datasource.ts#L135. As mentioned in PR description, I want to create a separate PR for adding node graph response processing to backend and/or update frontend processing to work with data frames instead of zipkin response.